### DISCUSSION TITLE
Granular window and tab state options

### DISCUSSION THREAD

**Discussion by danthegoodman1** [ghostty-org/ghostty#3756] [Feature Requests, Ideas] [answered]
Right now when ever I make a split or new tab, it uses the current dir.

I'd like to be able to customize like how I had in iterm2:
<img width="472" alt="image" src="https://github.com/user-attachments/assets/fbf991e0-85ed-445a-b81f-8778aea18211" />

**Comment by JakeWharton** [comment, marked as answer]
Already an accepted issue #1392
